modules/controller/dc-controller.js

```diff
@@ -41,31 +42,41 @@ class DCController {
                 if (!litigationIntervalInMinutes) {
                     litigationIntervalInMinutes = this.config.dc_litigation_interval_in_minutes;
                 }
+                const inserted_object = await Models.handler_ids.create({
+                    status: 'PENDING',
 
+                });
+                handlerId = inserted_object.dataValues.handler_id;
+                const offerId = await this.dcService.createOffer(
+                    req.body.dataset_id, dataset.root_hash, req.body.holding_time_in_minutes,
+                    req.body.token_amount_per_holder, dataset.otjson_size_in_bytes,
+                    litigationIntervalInMinutes, handlerId,
+                    req.body.urgent,
+                );
                 const handler_data = {
                     holding_time_in_minutes: req.body.holding_time_in_minutes,
                     token_amount_per_holder: req.body.token_amount_per_holder,
                     status: 'PUBLISHING_TO_BLOCKCHAIN',
```

Comment:
Please remove the status from the data, since the response already contains the status. Especially since the status inside the data is not updated upon errors

Reply:
This status if different then status that is in response.